### ISSUE TITLE
Documentation Issues - November-December 2015

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EnergyPlus#5313]
- [x] Refrigeration Documentation - bullet indenting causes confusion

For Group-Refrigeration, the InputOutputReference for EnergyPlus 8.4 shows under the "Detailed approach" all bullet items at the same level. Some of these bullet items should be indented further indicating what options may be used. Look at the EnergyPlus 8.2 to see how it is supposed to appear.

_See I/O Ref pp. 1485-1486 for corrections._

**Comment by mjwitte** [comment]
- [x] Engineering Reference, v8.4.0 pdf p. 738, section 16.2.24.8 Secondary Coils of DX Systems and Heat Pumps is at the wrong heading level - needs to come up one higher level.

_See Engineering Ref p. 926 for corrections._

**Comment by mjwitte** [comment]
- [x] Engineering Reference, v8.4.0 pdf p. 741, section 16.2.24.12 Sensible and Latent Split, some of the equations are not rendered.  The entire Secondary Coils section has lots of formatting problems.

_See Engineering Ref Pp. 927-930 for corrections. Equations on pp. 929&930 have been modified to match the source code [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/DXCoils.cc#L14354-L14362)_

```
                    // Calculate full load output conditions
                    FullLoadOutAirEnth = EvapInletEnthalpy - ( TotalHeatRemovalRate / PartLoadRatio ) / EvapAirMassFlow;
                    hTinwout = EvapInletEnthalpy - ( 1.0 - SHR ) * ( ( TotalHeatRemovalRate / PartLoadRatio ) / EvapAirMassFlow );
                    FullLoadOutAirHumRat = PsyWFnTdbH( EvapInletDryBulb, hTinwout, RoutineName, true );
                    FullLoadOutAirTemp = PsyTdbFnHW( FullLoadOutAirEnth, FullLoadOutAirHumRat );
                    // when the air outlet temperature falls below the saturation temperature, it is reset to saturation temperature
                    if ( FullLoadOutAirTemp < PsyTsatFnHPb( FullLoadOutAirEnth, OutBaroPress, RoutineName ) ) {
                        FullLoadOutAirTemp = PsyTsatFnHPb( FullLoadOutAirEnth, OutBaroPress, RoutineName );
                        FullLoadOutAirHumRat = PsyWFnTdbH( FullLoadOutAirTemp, FullLoadOutAirEnth, RoutineName );
```

**Comment by mjwitte** [comment]
- [x] Engineering Reference, v8.4.0 pdf p. 396, section 10.3.3, in the bullet for "DOAS Control Strategy = ColdSupplyAir" the last sentence says "If it is above T_l, it is heated to T_l."  Should this say cooled to T_l?

_See Engineering Ref p. 500 for corrections._

**Comment by mjwitte** [comment]
- [x] I/O Ref description for AirflowNetwork:MultiZone:Component:DetailedOpening still says
  "The AirflowNetwork model does not have a model for bi-directional flow through large horizontal openings. For this reason,
  AirflowNetwork:MultiZone:Component:DetailedOpening should not be used for horizontal openings. The best modeling technique in this case is to put an AirflowNetwork:MultiZone:Surface:Crack object in a horizontal surface and use a large air mass flow coefficient. Crack flow is assumed to be uni-directional in any given timestep (but can reverse flow direction from timestep to timestep)."

This should be replaced with:
"For large horizontal openings, use AirflowNetwork:MultiZone:Component:HorizontalOpening."

@lgu1234 please confirm.
Helpdesk ticket 10961

_See I/O Ref pp. 1075 for corrections._

**Comment by lgu1234** [comment]
@mjwitte It is partially true, because the AirflowNetwork:MultiZone:Component:HorizontalOpening is restricted to be used as an indoor partition, not exposed to outdoors right now. When a large horizontal opening is exposed outdoors, the above method is still valid.

**Comment by JasonGlazer** [comment]
- [x] Came across a small documentation issue related to ShadowCalculation. The CalculationMethod field is shown to have an option "DetailedTimestepIntegration" in the IOref but the IDD shows that option as "TimestepFrequency". 

_See I/O Ref p. 54 for corrections._

**Comment by Myoldmopar** [comment]
Closed via #5801
